app/js/lib/foxbox/services/light.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'light';
+
+export default class LightService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  // Public getter only properties.
+  get type() {
+    return TYPE;
+  }
+
+  // Public methods.
+  isAvailable() {
+    return this.get('available')
+      .then((response) => response[Object.keys(response)[0]].OnOff === 'On');
+  }
+
+  isOn() {
+    return this.get('LightOn')
+      .then((response) => response[Object.keys(response)[0]].OnOff === 'On');
+  }
+
+  /**
+   * Turn the bulb on or off.
+   *
+   * @param {boolean} on Whether to turn it on (true) or off (false).
+   * @return {Promise}
+   */
+  turn(on) {
+    const value = on ? 'On' : 'Off';
+    return this.set('LightOn', value);
+  }
+
+  /**
+   * Turn on the bulb.
+   *
+   * @return {Promise}
+   */
+  turnOn() {
+    return this.set('LightOn', 'On');
```

Comment:
__question:__ We don't use these methods anywhere yet, maybe let's keep it simple and add new methods only when we have such a need?

__nit:__ In case you decide to leave this method, it would be better to call ```this.turn(true)``` and ```this.turn(false)``` inside so that only one method has this unique and custom ```LightOn``` string.